### ISSUE TITLE
Make an AI Chatbot

### ISSUE THREAD

**Issue by Harshal0902** [Harshal0902/Rudra#6]
Make an AI chatbot assistant for each section.

**Comment by abhinav5481** [comment]
Hi Harshal,
I would like to work on this issue. Could you please assign it to me?

**Comment by Harshal0902** [comment]
@abhinav5481 you can work on this issue. You are requested to join the Discord server using this [link](https://discord.gg/qxgsn99W). Try to complete it within 5 days and follow the CONTRIBUTION.md for making PR.

**Comment by Harshal0902** [comment]
@abhinav5481 what's the status?
Are you facing any problem?

**Comment by abhinav5481** [comment]
I am about to complete it. Please give me some more time. I would create PR as soon as possible.

**Comment by Harshal0902** [comment]
ok, try to complete it ASAP

**Comment by unnati914** [comment]
i can make it. could i be assigned this issue?

**Comment by Harshal0902** [comment]
@unnati914 go for it.

**Comment by unnati914** [comment]
what features do you want for chatbot?

**Comment by Harshal0902** [comment]
The chatbot can help police/public with their questions. You can refer to chatbot.js for the services, and if you wish you can add more relevant features.

**Comment by unnati914** [comment]
sure thing

**Comment by PaliwalUmesh** [comment]
we can try google dialogue flow for this

**Comment by Harshal0902** [comment]
We can try dialogue flow, but there is a problem, in order to make an agent, you need to have a Google CLoud account, and you need to share your card details, in order to make an account.

**Comment by PaliwalUmesh** [comment]
ya that's really a problem 🤣.No worries we can always use python.

**Comment by PaliwalUmesh** [comment]
we can use chatterbot and make the AI chatbot. Is anybody working on it??

**Comment by Harshal0902** [comment]
@PaliwalRP can you please elaborate more about it

**Comment by PaliwalUmesh** [comment]
Actually the chatterbot is a python library which can be used to make a AI chatbot @Harshal0902

**Comment by PaliwalUmesh** [comment]
https://www.tutorialdocs.com/tutorial/chatterbot/simple-example.html#:~:text=Creating%20your%20first%20chat%20bot%201%20Setting%20the,bot.%20...%205%20Training%20your%20chat%20bot.%20 refer to this for more info.

**Comment by PaliwalRP** [comment]
@Harshal0902 we can create chatbot , whats the progress??

**Comment by Harshal0902** [comment]
Yes, we can do it, but we have to find a way to integrate it in our appliction

**Comment by NidhiChaurasia** [comment]
@Harshal0902 I can make an AI Chatbot using Dialogflow As I have made one for my previous hackathon.So,if you could assign me this issue I could contribute more!

Kindly check my COVID-19 Bot and assign me this issue :- 

https://bot.dialogflow.com/53e898d9-a118-4378-908f-177b32f5b34f

**Comment by Harshal0902** [comment]
@NidhiChaurasia you can make it using Dialogflow, but not sure it will work after free trial of Google Cloud is expire, so I will suggest to avoid it using Dialogflow. And we can afterwards make other modification with that chatbot like: voice command to open any page etc.
